payments-core/src/main/java/com/stripe/android/networking/PaymentAnalyticsRequestFactory.kt

```diff
@@ -314,37 +261,10 @@ class PaymentAnalyticsRequestFactory @VisibleForTesting internal constructor(
     }
 
     internal companion object {
-        internal const val FIELD_PRODUCT_USAGE = "product_usage"
-        internal const val FIELD_ANALYTICS_UA = "analytics_ua"
-        internal const val FIELD_APP_NAME = "app_name"
-        internal const val FIELD_APP_VERSION = "app_version"
-        internal const val FIELD_BINDINGS_VERSION = "bindings_version"
-        internal const val FIELD_IS_DEVELOPMENT = "is_development"
+        internal const val FIELD_TOKEN_TYPE = "token_type"
         internal const val FIELD_DEVICE_ID = "device_id"
-        internal const val FIELD_DEVICE_TYPE = "device_type"
-        internal const val FIELD_EVENT = "event"
-        internal const val FIELD_OS_NAME = "os_name"
-        internal const val FIELD_OS_RELEASE = "os_release"
-        internal const val FIELD_OS_VERSION = "os_version"
-        internal const val FIELD_PUBLISHABLE_KEY = "publishable_key"
+        internal const val FIELD_PRODUCT_USAGE = "product_usage"
         internal const val FIELD_SOURCE_TYPE = "source_type"
         internal const val FIELD_3DS2_UI_TYPE = "3ds2_ui_type"
-        internal const val FIELD_TOKEN_TYPE = "token_type"
-
-        @JvmSynthetic
-        internal val VALID_PARAM_FIELDS: Set<String> = setOf(
-            FIELD_ANALYTICS_UA, FIELD_APP_NAME, FIELD_APP_VERSION, FIELD_BINDINGS_VERSION,
-            FIELD_IS_DEVELOPMENT, FIELD_DEVICE_TYPE, FIELD_EVENT, FIELD_OS_VERSION, FIELD_OS_NAME,
-            FIELD_OS_RELEASE, FIELD_PRODUCT_USAGE, FIELD_PUBLISHABLE_KEY, FIELD_SOURCE_TYPE,
-            FIELD_TOKEN_TYPE
-        )
```

Comment:
This seems to be used just for testing purposes, so moved it to `PaymentsAnalyticsFactoryTest`.